src/auth.coffee

```diff
@@ -9,7 +9,7 @@ catch error
   else
     throw error
 
-tokenName = 'Atom.io API Token'
+tokenName = 'pulsar-edit.dev API Token'
```

Comment:
⚠️ This and the next one will probably require users logging back into the Pulsar package API, or else copying over their token they added when this used/overwrote the Atom API token. We may get support requests over this.

I think it is the right thing to do, to not clobber the existing Atom token, but something to be aware of.

Reply:
We have an opportunity to potentially clarify that this is the  "pulsar-edit.dev **Package** API Token", as opposed to the somewhat similarly named `atom-github` token for the `github` package. (Maybe we will rename this token to `pulsar-github` some day... ?).

So if anyone agrees with this suggestion, then feel free to update this tokenName to say "pulsar-edit.dev **Package** API Token". If no-one agrees then, LGTM as-is. Not a blocker IMO.

Reply:
I'll bow to your wisdom on this one to be honest so happy to make the change there (and it seems to have @confused-Techie's blessing too)